libfreerdp/core/listener.c

```diff
@@ -64,6 +127,16 @@ static BOOL freerdp_listener_open(freerdp_listener* instance, const char* bind_a
 	if (!bind_address)
 		ai_flags = AI_PASSIVE;
 
+#ifdef AF_VSOCK
+	const char* vsockPrefix = "vsock://";
```

Comment:
maybe a good idea to move this to a function (you use the check on more than one occation)